docs/guides/component-testing/getting-started.mdx

```diff
@@ -0,0 +1,622 @@
+[Qwik](https://github.com/qwikifiers/cypress-qwik) and [Lit](https://github.com/redfox-mx/cypress-lit).
+
+### Your First Component Test
+
+Though every framework is different and have their own unique nuances, in general, writing tests are very similar.
+Let's look at how a basic test is written for a `StepperComponent`.
+
+:::note
+
+Let's assume the Stepper Component consists of two `buttons`, one used to **decrement** the counter and one
+to **increment** it as well as `span` tag sits in the middle of the buttons to show the
+current value of the counter.
+
+:::
+
+<Tabs>
+<TabItem value="React" active>
+
+```jsx
+import React from 'react'
+import Stepper from './Stepper'
+
+describe('<Stepper />', () => {
+  it('mounts', () => {
+    // see: https://on.cypress.io/mounting-react
+    cy.mount(<Stepper />)
+  })
+})
+```
+
+</TabItem>
+<TabItem value="Angular">
+
+```typescript
+import { StepperComponent } from './stepper.component'
+
+describe('StepperComponent', () => {
+  it('mounts', () => {
+    // see: https://on.cypress.io/mounting-angular
+    cy.mount(StepperComponent)
+  })
+})
+```
+
+</TabItem>
+<TabItem value="Vue">
+
+```js
+import Stepper from './Stepper.vue'
+
+describe('<Stepper />', () => {
+  it('mounts', () => {
+    // see: https://on.cypress.io/mounting-vue
+    cy.mount(Stepper)
+  })
+})
+```
+
+</TabItem>
+<TabItem value="Svelte">
+
+```js
+import Stepper from './Stepper.svelte'
+
+describe('Stepper', () => {
+  it('mounts', () => {
+    // see: https://on.cypress.io/mounting-svelte
+    cy.mount(Stepper)
+  })
+})
+```
+
+</TabItem>
+</Tabs>
+
+### Test Explanation
+
+Let's break down the spec. First, we import the `Stepper` component. Then, we
+organize our tests using the functions `describe` and `it`, which allows us to
+group tests into sections by using method blocks. These are global functions
+provided by Cypress, which means you don't have to import them directly to use
+them. The top-level `describe` block will be the container for all our tests in
+a file, and each `it` represents an individual test. The `describe` function
+takes two parameters, the first of which is the name of the test suite, and the
+second is a function that will execute the tests.
+
+We defined a test using the `it` function inside `describe`. The first parameter
+to `it` is a brief description of the spec, and the second parameter is a
+function that contains the test code. In our example above, we only have one
+test, but soon we'll see how we can add multiple `it` blocks inside of a
+`describe` for a series of tests.
+
+:::info
+
+The [cy.mount()](/api/commands/mount) method will mount our component into the test
+app so we can begin running tests against it.
+
+:::
+
+Now it's time to see the test in action.
+
+## Running the Test
+
+Switch back to the browser you opened for testing, and you should now see the
+**Stepper** Component in the spec list. Click it to see the spec execute.
```

Comment:
Maybe a screenshot would help here to show the item in the list, we haven't actually shown the spec list yet (except for in the video).

Reply:
Added a quick video to show this